doc/migrating-configs.md

```diff
@@ -2,544 +2,130 @@
-        }
-      }
-    ]
-  }
-}
-```
-
-This `create` object has been deprecated. Configurations of version 2.1.0 that use the `create` object will still work, but will cause Ignition to log a warning.
-
-The fields that existed in the `create` object have been added directly to the `users` object, and it's advised to use these new fields instead of the `create` object.
-
-```json ignition
-{
-  "ignition": {
-    "version": "2.1.0"
-  },
-  "passwd": {
-    "users": [
-      {
-        "name": "test",
-        "uid": 1010,
-        "gecos": "user creation test",
-        "noCreateHome": true,
-        "noUserGroup": true
-      }
-    ]
-  }
-}
-```
-
-## From Version 1 to 2.0.0
-
-This section will cover the breaking changes made between versions 1 and 2.0.0 of the configuration specification.
-
-### Version
-
-One of the more notable changes was the representation of the config version, moving from an integer to a [Semantic Version][semver] string. Using a Semantic Version will allow the configuration specification to pick up additions and other backward-compatible changes in the future without necessarily requiring the user to update their config. The version number has also moved locations and is now in an Ignition metadata section named "ignition".
-
-The following shows the changes to the version section:
-
-```json ignition
-{
-  "ignitionVersion": 1
-}
-```
-
-```json ignition
-{
-  "ignition": {
-    "version": "2.0.0"
-  }
-}
-```
-
-### Files
-
-The `files` section was moved out from under `filesystems` and is now directly under the `storage` section. This was done in order to decouple file definitions from filesystem definitions. This is particularly useful when merging multiple configs together. One config may define a filesystem while another can write files to that filesystem without needing to know the specifics of that filesystem. A common example of this is referencing the universally-defined "root" filesystem which is defined by default inside of Ignition.
-
-The following shows this particular change to the files section:
-
-```json ignition
-{
-  "ignitionVersion": 1,
-  "storage": {
-    "filesystems": [
-      {
-        "device": "/dev/sdb1",
-        "format": "ext4",
-        "files": [
-          {
-            "path": "/foo/bar"
-          }
-        ]
-      }
-    ]
-  }
-}
-```
-
 ```json ignition
 {
-  "ignition": {
-    "version": "2.0.0"
-  },
+  "ignition": { "version": "3.0.0" },
   "storage": {
-    "filesystems": [
-      {
-        "name": "example",
-        "mount": {
-          "device": "/dev/sdb1",
-          "format": "ext4"
-	}
-      }
-    ],
-    "files": [
-      {
-        "filesystem": "example",
-        "path": "/foo/bar"
-      }
-    ]
+    "files": [{
+      "path": "/example-file",
+      "mode": 420,
+      "append": [
+        { "source": "data:,hello" },
+        { "source": "data:,world" }
+      ]
+    }]
   }
 }
 ```
 
-#### Contents
+### File permissions now default to 0644
```

Comment:
Should we move this section to be directly after the section mentioning directory defaults?